code/modules/antagonists/vampire/vampire_powers/gargantua_powers.dm

```diff
@@ -201,3 +201,92 @@
 		var/mob/living/L = user
 		L.apply_status_effect(STATUS_EFFECT_CHARGING)
 		L.throw_at(target, targeting.range, 1, L, FALSE, callback = CALLBACK(L, TYPE_PROC_REF(/mob/living, remove_status_effect), STATUS_EFFECT_CHARGING))
+
+#define ARENA_SIZE 3
+/obj/effect/proc_holder/spell/vampire/arena
+	name = "Desecrated Duel (150)"
+	desc = "You leap towards someone. Upon landing, you conjure an arena, and within it you will heal brute and burn damage, recover from fatigue faster, and be strengthened against lasting damages. Can be recasted to end the spell early."
+	gain_desc = "You can now leap to a target, creating an arena where you land."
+	required_blood = 150
+	base_cooldown = 30 SECONDS
+	action_icon_state = "duel"
+	should_recharge_after_cast = FALSE
+	/// Is our spell active?
```

Comment:
is this comment really necessary? `spell_active` is a pretty descriptive var name

Reply:
Force of habit haha